test/distributed/_tensor/test_matrix_ops.py

```diff
@@ -120,6 +120,25 @@ def test_placement_comb(
         for spec in shard_specs_comb:
             test_placement_comb([spec[0]], [spec[1]])
 
+    @with_comms
+    def test_matmul(self):
+        device_mesh = DeviceMesh(self.device_type, list(range(self.world_size)))
+        dim = 128
+        x = torch.randn(8, dim)
+        A = torch.randn(dim, dim)
+        y = torch.matmul(x, A)
+
+        # Prepare DTensors
+        dx = distribute_tensor(x, device_mesh, [Replicate()])
+        dA = distribute_tensor(A, device_mesh, [Shard(0)])
```

Comment:
i remember that people complain the var name `dX` `dA` indicates gradients. Names such as `x_dist` `A_dist`are preferred. cc @awgu